### PR TITLE
Update split voting 

### PR DESCRIPTION
**Pull request by Subham2804**

### CODE REVIEW

File: packages/.DS_Store
Comment (odegnome) [inline packages/.DS_Store:1]: Remove this file!

Comment (Subham2804) [reply packages/.DS_Store:1]: resolved

File: src/contract.rs [hunk 1]
```diff
@@ -1239,93 +1409,69 @@ pub fn vote_proposal(
...
-                extended_pair: extended_pair_param,
-                vote_weight: vote_power,
-            };
-
-            VOTERSPROPOSAL.save(deps.storage, (info.sender.clone(), proposal_id), &vote)?;
-        } else {
-            let mut prev_proposal_vote = PROPOSALVOTE
-                .load(deps.storage, (proposal_id, last_voted_pair))
+        let last_vote_weight = prev_vote.votes;
+        for pair_vote in last_vote_weight {
```
Comment (odegnome) [inline src/contract.rs:1429]: All previous votes are being removed even if they are not being updated. Is this desired?

Comment (Subham2804) [reply src/contract.rs:1429]: votersproposals is overridden later on in the function. prev vote is just used here to decrease individual pair vote weight. so it is expected

**Review by odegnome — APPROVED** [review]

**Pull request merged**